persistence/jdbc/src/test/java/org/infinispan/test/fwk/UnitTestDatabaseManager.java

```diff
@@ -144,7 +144,7 @@ public static void verifyConnectionLeaks(ConnectionFactory connectionFactory) {
          PooledConnectionFactory pcf = (PooledConnectionFactory) connectionFactory;
          try {
             Thread.sleep(500); // C3P0 needs a little delay before reporting the correct number of connections. Bah!
```

Comment:
@slaskawi This is code was included as part of ISPN-2033 and I assumed that it was a necessary evil for C3P0 to play nice. I could try and change this, however this code will be removed in a future release when C3P0 support is removed.